### ISSUE TITLE
Enable nftables support in kernel / module

### ISSUE THREAD

**Issue by felixsanz** [coreos/bugs#1421]
# Issue Report
## Feature Request

nf_tables module or kernel support for nftables.
### Environment

All
### Desired Feature

nftables
### Other Information

Reproduce in toolbox:
1. dnf install nftables
2. nft -f /etc/nftables/ipv4-filter # Error

Check:

```
modprobe nf_tables
modprobe: FATAL: Module nf_tables not found.
```

**Comment by TerraTech** [comment]
@crawford would this include the 'nft' binary in addition to enabling it in the kernel?

**Comment by crawford** [comment]
We could probably include `nft`, but I'm not certain yet.

**Comment by sanmai-NL** [comment]
@crawford: What factors hold this up or need to be cleared up?

**Comment by crawford** [comment]
We just need to take a look at the `nft` binary and make sure it's not doing anything crazy. It's very unlikely that anything will come up, but we have to take a look (and haven't had time yet).

**Comment by sanmai-NL** [comment]
@crawford: can anyone help you with this process?

**Comment by crawford** [comment]
@sanmai-NL this is going to be up to @coreos/team-os-internal-container-linux.

**Comment by bgilbert** [comment]
This is done in https://github.com/coreos/coreos-overlay/pull/2662 and should be included in 1492.0.0.  For now, we'll be shipping the `nft` binary and the default rulesets (the latter in `/usr/share/nftables`) but no automation to save rules at shutdown or restore them at startup.
